src/components/Phonescatalog/Phonescatalog.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { FC, useState, useEffect, useMemo } from 'react';
+
+import { URL_PHONE } from '../constants/api';
+
+import '../Style/card.scss';
+
+export const Phonescatalog: FC = () => {
+  const [phones, setPhones] = useState<Phone[]>([]);
+
+  useEffect(() => {
+    fetch(URL_PHONE)
+      .then(async data => setPhones(await data.json()));
+  }, []);
+
+  const phonesShow = useMemo(() => {
+    if (!phones.length) {
+      return [];
+    }
+
+    return phones;
+  }, [phones]);
+
+  return (
+    <main className="main">
+      <section className="wrapper">
+        <h1>Phones page))</h1>
```

Comment:
давай будем серьезно подходить к проектам портфолио
```suggestion
        <h1>Phones page</h1>
```